hardhat.config.ts

```diff
@@ -57,7 +57,7 @@ const hardhatConfig: HardhatUserConfig = {
   },
   paths: {
     sources: "./contracts",
-    tests: "./test",
+    tests: "./test/rescue",
```

Comment:
does this affect our other `tests`?

Reply:
Yes, but in this branch, I only want to test the rescue test script

Reply:
So I guess this needs to be removed before we merge it to main?

Reply:
Added `TEST_WHITEHAT_ONLY` flag in `.env` to decide which tests to run. By default, only run tests for "Stake" contract.